README.md

```diff
@@ -31,29 +44,33 @@ Example with docker-compose:
 ```yaml
 
 database:
-  image: postgres:12.1
-  volumes:
-    - ./persistent/data:/var/lib/postgresql/data
-  environment:
-    - POSTGRES_PASSWORD=SumPassw0rdHere
-    - POSTGRES_DB=postgres
-  restart: unless-stopped
-
-postgres-backup:
```

Comment:
Can you make two example docker-compose.yml files, one with a single combined dump/pg and one with them separate?  I like your enterprise on having fewer services, but I prefer to keep my services simple and task focused.

Reply:
I've created a second file named `docker-compose-double.yml` and included the contents in the README